test/integration/users.test.js

```diff
@@ -350,6 +351,89 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users?search", function () {
+    beforeEach(async function () {
+      await addOrUpdate(userData[0]);
+      await addOrUpdate(userData[7]);
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+    });
+
+    it("Should return users successfully", function (done) {
+      chai
+        .request(app)
+        .get("/users")
+        .query({ search: searchParamValues[0] })
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Users returned successfully!");
+          expect(res.body.users).to.be.a("array");
+
+          return done();
+        });
+    });
+    it("Should return users successfully converting search param value to small case", function (done) {
+      chai
+        .request(app)
+        .get("/users")
+        .query({ search: searchParamValues[1] })
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Users returned successfully!");
+          expect(res.body.users).to.be.a("array");
+
+          return done();
```

Comment:
Here only checking if array or not, 
Can we check, if correct data is being returned

Reply:
done